frontend/src/scenes/organization/TeamMembers/CreateInviteModal.tsx

```diff
@@ -44,28 +44,17 @@ export function CreateInviteModalWithButton({ type = 'standalone' }: { type?: 's
 
     return (
         <>
-            {type === 'sidebar' ? (
-                <span
-                    className="sidebar-label"
-                    onClick={() => {
-                        setIsVisible(true)
-                    }}
-                >
-                    Invite Team Member
-                </span>
-            ) : (
-                <Button
-                    type="primary"
-                    data-attr="invite-teammate-button"
-                    onClick={() => {
-                        setIsVisible(true)
-                    }}
-                    icon={<PlusOutlined />}
-                >
-                    Invite Team Member
-                </Button>
-            )}
-
+            <Button
+                type="primary"
+                data-attr="invite-teammate-button"
+                onClick={() => {
+                    setIsVisible(true)
+                }}
+                icon={<PlusOutlined />}
```

Comment:
There's a marginally nicer icon `<UserAddOutlined />`

Reply:
Oh, definitely, that's a good one